src/OSPSuite.Core/Comparison/BuildingBlockDiffBuilder.cs

```diff
@@ -38,11 +38,21 @@ public MoleculeBuildingBlockDiffBuilder(ObjectBaseDiffBuilder objectBaseDiffBuil
       }
    }
 
-   public class SpatialStructureDiffBuilder : BuildingBlockDiffBuilder<ISpatialStructure, IContainer>
+   public class SpatialStructureDiffBuilder : DiffBuilder<ISpatialStructure>
```

Comment:
Because of the IBuilder requirements (and because I did not want all container to implement this interface, I had to create a special implementation for Spatial Structure)